config/install/field.storage.paragraph.field_oe_icon.yml

```diff
@@ -0,0 +1,191 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - options
+    - paragraphs
+id: paragraph.field_oe_icon
+field_name: field_oe_icon
+entity_type: paragraph
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: arrow-down
+      label: arrow-down
```

Comment:
Change labels into "human readable labels", so capitalized and without any dash, for example:
`label: arrow-down` should become `label: "Arrow down"`.